src/test/java/kz/aidyninho/service/TicketServiceTest.java

```diff
@@ -0,0 +1,108 @@
+package kz.aidyninho.service;
+
+import kz.aidyninho.dto.TicketDto;
+import kz.aidyninho.model.Ticket;
+import kz.aidyninho.model.Type;
+import kz.aidyninho.model.User;
+import kz.aidyninho.repository.TicketRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.io.File;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class TicketServiceTest {
+
+    private static final Ticket TICKET = new Ticket(1L, new User(), Type.DAY, LocalDate.of(2024, 1, 1));
+    @Mock
+    private TicketRepository ticketRepository;
+
+    @InjectMocks
+    private TicketService ticketService;
+
+    @Test
+    public void findById() {
```

Comment:
This is not proper name for test, the proper one should ne something like findById_And_Ticket_Returned_Successful
Also you should not test case when error thrown in same test and extract it to separated one